definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -13942,5 +13942,75 @@
     "CaseFieldID": "DateCoRespReceivedDivorceApplication",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "20/09/2019",
```

Comment:
can you set `LiveFrom` to` today` date  which can help us to know roughly when this field is introduced

Reply:
Done